flask_discord_interactions/enums/__init__.py

```diff
@@ -0,0 +1,5 @@
+from flask_discord_interactions.enums.response_types import ResponseType
+
+__all__ = [
+    "ResponseType",
```

Comment:
I'm a bit hesitant to introduce an entirely new sub-package just for a single 7-line enum, but I do see the value of this going forward. I suppose the cleaner code is probably worth the complexity.

Reply:
We probably should move it to somewhere in the models, but I'm not sure which file would be the most appropriate (or create a new file in there). That said, if you're fine with the folder, it might be OK to keep. I don't have much experience designing APIs or long projects though.

Reply:
We could do something like `models/response.py`, and have a superclass (say, `Response`) of both `Message` and `Modal`. It would be nice, semantically, for every command function to return some type of `Response` object. That said, I don't think they have a ton of logic in common.

Reply:
I haven't really thought of a better way to do this, so the enum directory can stay for now.